bottlecap/src/lifecycle/invocation_times.rs

```diff
@@ -40,22 +47,23 @@ impl InvocationTimes {
         let should_adapt = (elapsed as f64 / (LOOKBACK_COUNT - 1) as f64) < ONE_TWENTY_SECONDS;
         if should_adapt {
             // Both units here are in seconds
+            // TODO: What does this mean?
```

Comment:
Need help here

Reply:
Sorry if I was not clear. What I don't understand is why compare `elapsed` with `flush_timeout`.

For example, if `flush_timeout` is 30s, there's one invocation every 1.6s, then `elapsed` will be 1.6s * 19 = 30.4s, and we will choose the synchronous "periodic" strategy with 20s interval. If flush takes 29s, then this will block lots of invocations.

Should we compare the average invocation interval (`elapsed / (LOOKBACK_COUNT - 1)`) with `flush_timeout` instead?